freqtrade/configuration/__init__.py

```diff
@@ -0,0 +1,8 @@
+from freqtrade.configuration.arguments import Arguments, TimeRange  # noqa: F401
+
+from freqtrade.configuration.arguments import (  # noqa: F401
```

Comment:
I wouldn't copy/import these constants here - we can also use direct imports from the few places we need them ...

Reply:
Agree, done.